### PR TITLE
Add calculated fields

### PR DESCRIPTION
**Pull request by FinnStutzenstein**

### CODE REVIEW

File: docs/models.txt [hunk 1]
```diff
@@ -306,6 +306,7 @@ Interface meeting {
     present_user_ids: (user/is_present_in_meeting_ids)[];
     temporary_user_ids: (user/meeting_id)[];
     guest_ids: (user/guest_meeting_ids)[];
+    user_ids: (Id)[]; // Calculated: All ids from temporary_user_ids, guest_ids and all users assigned to groups.
```
Comment (FinnStutzenstein) [inline docs/models.txt:309]: @normanjaeckel Should this get a reverse relation? In general: Should calculated fields get a reverse relation (if possible)?

File: docs/models.txt [hunk 2]
```diff
@@ -425,6 +426,7 @@ Interface motion {
     sort_child_ids: (motion/parent_id)[];
     origin_id: motion/derived_motion_ids; // Note: The related motions may not be in the same meeting
     derived_motion_ids: (motion/origin_id)[]; // Note: The related motions may not be in the same meeting
+    forwarding_tree_motion_ids: (Id)[]; // Calculated: All children (derived_motion_ids), grand children, ... and all parents (origin_id).
```
Comment (FinnStutzenstein) [inline docs/models.txt:429]: Same here.

File: docs/models.txt (hunk 1 above)
Comment (normanjaeckel) [reply docs/models.txt:310]: I hope not ...

Comment (FinnStutzenstein) [reply docs/models.txt:310]: I currently also do not see the advantage, so for me it is ok.

**Pull request merged**